### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by chris-luangrath**
Pull request template

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Pull Request Checklist
```
Comment (vinhowe) [inline pull_request_template.md:1]: Can you rewrite this in active voice? "I wrote tests," "My code passed," etc.

**Review by vinhowe — CHANGES_REQUESTED** [review]

**Comment by vinhowe** [comment]
LGTM. @daniekpo, what do you think?

**Review by daniekpo — APPROVED** [review]
LGTM

**Comment by daniekpo** [comment]
> LGTM. @daniekpo, what do you think?

Looks good to me.

**Review by vinhowe — APPROVED** [review]

**Pull request merged**